app/controllers/request_game_controller.rb

```diff
@@ -25,9 +25,11 @@ def play
               order('random()')
 
     if @missing == 0
-      flash[:notice] = _('<p>All done! Thank you very much for your help.</p><p>There are <a href="{{helpus_url}}">more things you can do</a> to help {{site_name}}.</p>',
-                         :helpus_url => help_credits_path(:anchor => "helpus"),
-                         :site_name => site_name)
+      flash[:notice] = {:partial => "game_over.html.erb",
```

Comment:
This should be `flash.now[:notice]` too I think , because there's not redirect afterwards, I added a commit but I think the specs will need a tweak too, because IIRC it's harder to test flash.now in controller specs?